cabby/geo/map_processing/map_processor.py

```diff
@@ -40,17 +40,16 @@
 
 
 def main(argv):
-    del argv  # Unused.
-    map = map_structure.Map(FLAGS.region, FLAGS.min_s2_level)
+  del argv  # Unused.
+  map = map_structure.Map(FLAGS.region, FLAGS.min_s2_level)
 
-    # Write to disk.
-    if FLAGS.directory is not None:
-      map.write_map(FLAGS.directory)
+  # Write to disk.
+  map.write_map(FLAGS.directory)
```

Comment:
Need to mark 'directory' as required since you no longer check if it is not None?